Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 31345)
+++ CMakeLists.txt	(working copy)
@@ -2,14 +2,9 @@
 cmake_minimum_required (VERSION 2.6)
 
 set(CMAKE_VERBOSE_MAKEFILE TRUE)
-set(VERSION "1.7.18")
+set(VERSION "1.11.1")
 
-# check for 64 bit
-if (CMAKE_SIZEOF_VOID_P EQUAL 8)
-  set(LIB_INSTALL_DIR "lib64")
-else()
-  set(LIB_INSTALL_DIR "lib")
-endif()
+add_definitions(-DYAT_PROJECT_VERSION=${VERSION})
 
 if (UNIX)
   add_definitions(-std=c++0x -Wall -Wextra)
@@ -21,5 +16,5 @@
 if (UNIX)
   configure_file("${CMAKE_CURRENT_SOURCE_DIR}/yat.pc.in"
   "${CMAKE_CURRENT_BINARY_DIR}/yat.pc" @ONLY IMMEDIATE)
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/yat.pc DESTINATION ${LIB_INSTALL_DIR}/pkgconfig)
+  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/yat.pc DESTINATION lib/pkgconfig)
 endif()
Index: src/CMakeLists.txt
===================================================================
--- src/CMakeLists.txt	(revision 31345)
+++ src/CMakeLists.txt	(working copy)
@@ -1,5 +1,6 @@
 set (source_files
   Exception.cpp
+  Version.cpp
   bitsstream/BitsStream.cpp
   bitsstream/Endianness.cpp
   file/FileName.cpp
@@ -48,13 +49,13 @@
   ../include
 )
 
-add_library (yat SHARED ${source_files})
+add_library (yat ${source_files})
 
 if (WIN32)
 	target_link_libraries(yat ws2_32.lib shell32.lib)
 endif()
 
-install (DIRECTORY ../include/yat DESTINATION include/
+install (DIRECTORY ../include/yat DESTINATION include
   FILES_MATCHING PATTERN "*.h"
   PATTERN "*.tpp"
   PATTERN "*.i"
@@ -65,5 +66,8 @@
 )
 
 if (UNIX)
-	install (TARGETS yat LIBRARY DESTINATION ${LIB_INSTALL_DIR})
+	install(TARGETS yat
+	    ARCHIVE DESTINATION lib
+	    LIBRARY DESTINATION lib
+	    RUNTIME DESTINATION bin)
 endif()
